star-gateway/HIL_SIMULATION.md

```
@@ -0,0 +1,299 @@
+| Constant | Value | File | Description |
+|----------|-------|------|-------------|
+| `grpcListenPort` | `:50051` | `cmd/star-gateway/main.go` | gRPC port |
+| `httpListenPort` | `:8080` | `cmd/star-gateway/main.go` | HTTP/WebSocket port |
+| `httpShutdownTimeout` | 5 s | `cmd/star-gateway/main.go` | Shutdown timeout |
+| `grpcMaxMsgSize` | 10 MB | `cmd/star-gateway/main.go` | Max message size |
+
+## Design Principles
+
+This implementation follows strict coding standards:
+
+### SOLID Principles
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)